apps/condo/domains/organization/access/Organization.js

```diff
@@ -2,12 +2,13 @@
  * Generated by `createschema organization.Organization 'country:Select:ru,en; name:Text; description?:Text; avatar?:File; meta:Json; employees:Relationship:OrganizationEmployee:CASCADE; statusTransitions:Json; defaultEmployeeRoleStatusTransitions:Json' --force`
  */
 const { throwAuthenticationError } = require('@condo/keystone/apolloErrorFormatter')
-const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RESIDENT, SERVICE } = require('@condo/domains/user/constants/common')
 const { get, uniq, compact } = require('lodash')
 const access = require('@condo/keystone/access')
 const { find } = require('@condo/keystone/schema')
 
-async function canReadOrganizations ({ authentication: { item: user } }) {
+async function canReadOrganizations (params) {
+    const { authentication: { item: user } } = params
```

Comment:
Why can't you parse params in arguments?

Reply:
It was for debugging arguments ;-)
Restored